static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx

```diff
@@ -158,13 +163,26 @@ function SimilarStackTrace({params, location, project}: Props) {
             </EmptyStateWarning>
           </Panel>
         )}
-        {status === 'ready' && hasSimilarItems && (
+        {status === 'ready' && hasSimilarItems && !hasSimilarityEmbeddingsFeature && (
           <List
             items={items.similar}
             filteredItems={items.filtered}
             onMerge={handleMerge}
             orgId={orgId}
             project={project}
+            organization={organization}
+            groupId={groupId}
+            pageLinks={items.pageLinks}
+          />
+        )}
+        {status === 'ready' && hasSimilarItems && hasSimilarityEmbeddingsFeature && (
+          <List
+            items={items.similar.concat(items.filtered)}
+            filteredItems={[]}
```

Comment:
combine similar items and filtered items into `items` to show everything regardless of if it's below the threshold